### PR TITLE
improved slideshows handling using API

### PR DESCRIPTION
**Pull request by takeit**
- [x] - unlinking slideshow from the article
- [x] - improved getting single slideshow
- [x] - finish specs

### CODE REVIEW

**Comment by ahilles107** [comment]
fo behat tests we will need api to create slideshows, or we will somehow fill db with after installation and before tests

**Comment by ahilles107** [comment]
it's ready?

**Comment by takeit** [comment]
@ahilles107 I need to check specs if all is fine

**Comment by m038** [comment]
@takeit can you do a rebase for all your commits?

File: newscoop/src/Newscoop/NewscoopBundle/Resources/translations/articles.pl.yml [hunk 1]
```diff
@@ -189,7 +189,7 @@ Snippet: Fragment
 'List name should not be longer than 40 chars': 'Nazwa listy nie może być dłuższa niż 40 znaków'
 'Not valid number': 'Podana wartość nie jest liczbą'
 'List limit': 'Limit artykułów na liście'
-'The last article of this list has been removed.': null
+'The last article of this list has been removed.': 'Ostatni artykuł z tej listy został usunięty.'
```
Comment (m038) [inline newscoop/src/Newscoop/NewscoopBundle/Resources/translations/articles.pl.yml]: :100: this looks awesome!

**Comment by m038** [comment]
:+1

**Comment by takeit** [comment]
rebased

**Pull request merged**